src/dev/windows/lv_windows_context.c

```diff
@@ -0,0 +1,659 @@
+/**
+ * @file lv_windows_context.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_windows_context.h"
+#if LV_USE_WINDOWS
+
+#include "lv_windows_display.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+/**********************
+ *      TYPEDEFS
+ **********************/
+
+/**********************
+ *  STATIC PROTOTYPES
+ **********************/
+
+static void lv_windows_check_display_existence_timer_callback(
+    lv_timer_t * timer);
+
+static LRESULT CALLBACK lv_windows_window_message_callback(
+    HWND   hWnd,
+    UINT   uMsg,
+    WPARAM wParam,
+    LPARAM lParam);
+
+bool lv_windows_pointer_device_window_message_handler(
+    HWND hWnd,
+    UINT uMsg,
+    WPARAM wParam,
+    LPARAM lParam,
+    LRESULT * plResult);
+
+bool lv_windows_keypad_device_window_message_handler(
+    HWND hWnd,
+    UINT uMsg,
+    WPARAM wParam,
+    LPARAM lParam,
+    LRESULT * plResult);
+
+bool lv_windows_encoder_device_window_message_handler(
+    HWND hWnd,
+    UINT uMsg,
+    WPARAM wParam,
+    LPARAM lParam,
+    LRESULT * plResult);
+
+/**********************
+ *  STATIC VARIABLES
+ **********************/
+
+/**********************
+ *      MACROS
+ **********************/
+
+/**********************
+ *   GLOBAL FUNCTIONS
+ **********************/
+
+void lv_windows_platform_init(void)
+{
+    lv_timer_create(
```

Comment:
The returned timer pointer needs to be recorded, otherwise it will cause memory leaks